### PR TITLE
Updating

### PR DESCRIPTION
**Pull request by lucywyman**
* Updating schedule page
* Adding sponsors to footer
* Updating information page with current information (wifi, rooms, etc.)
* Adding draft of lightning talk blog post, with link on front page

### CODE REVIEW

File: barcamp-theme/templates/base.html [hunk 1]
```diff
@@ -61,7 +61,11 @@
 
 		    <p class="lead">Beaver BarCamp is an informal conference where everyone is encouraged to participate and the sessions are not predetermined. <a href="/pages/about.html"><em>Learn more</em></a></p>-->
 		    <h5>Event Sponsors</h5>
-        <p>Want to sponsor the event? Please contact info < at > beaverbarcamp.org</p>
+            <img class='sponsor' src='/theme/img/github.png' />
```
Comment (ramereth) [inline barcamp-theme/templates/base.html]: Remove all of these leading ``/``. When I build the site, it tries accessing from ``/theme`` which is not right. Leaving it out ensures its relative. Please make the change to the other images.

Comment (ramereth) [inline barcamp-theme/templates/base.html]: Also, most of these sponsors aren't returning this year. Only Mozilla so far and OSU.

File: barcamp-theme/templates/index.html [hunk 2]
```diff
@@ -2,12 +2,13 @@
...
 {% block content %}
     <div id="feature-text">
-        <h2>Share Your <br />Passion</h2>
+        <h2>Share Your Passion</h2>
         <div id="beaver-small"><img src="{{ SITEURL }}/theme/img/beaver-small.gif" alt="Beaver Barcamp icon" /></div>
         <h4>{{ SITENAME }}</h4>
-        <h5>April 18, 2015 at the<br />Kelley Engineering Center<br><br>
-        <p>Registration is free<br>Sign up today!</p>
-        <a href="http://www.eventbrite.com/e/beaver-barcamp-15-tickets-15983994558" id="register-button">Register</a></h5>
+        <h5>Now with <a href="/lightning-talks.html">Lightning Talks!</a></h5>
```
Comment (ramereth) [inline barcamp-theme/templates/index.html]: Convert to a relative link

File: content/pages/FAQ.rst [hunk 3]
```diff
@@ -15,7 +14,13 @@ How are topics chosen?
 Most session topics are scheduled the morning of the event and are provided by
 the attendees.
 
---------------
+What are Lightning Talks?
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Lightning talks are five minute presentations on any topic. At barcamp,
+the first session will be just lightning talks in the barcamp style;
```
Comment (ramereth) [inline content/pages/FAQ.rst]: This makes it sound like the only first session will be lightning talks. Please reword to clarify.

File: barcamp-theme/templates/index.html [hunk 4]
```diff
@@ -2,12 +2,13 @@
...
+        <h5>April 18, 2015 at the<br />Kelley Engineering Center<br>
+        <p>Registration is free-- sign up today!</p>
+        <a href="http://www.eventbrite.com/e/beaver-barcamp-15-tickets-15983994558" id="register-button">Register</a></h5><br><br>
```
Comment (ramereth) [inline barcamp-theme/templates/index.html]: Let's go ahead and use the shorter URL http://beaverbarcamp15.eventbrite.com

File: content/pages/FAQ.rst [hunk 5]
```diff
@@ -53,16 +55,14 @@ RSVP is not required, but we would love if you registered for the event on
 `EventBrite`_. This helps us plan for enough food and accommodate dietary
 restrictions.
 
-.. _EventBrite: http://beaverbarcamp14.eventbrite.com
+.. _EventBrite: http://www.eventbrite.com/e/beaver-barcamp-15-tickets-15983994558
```
Comment (ramereth) [inline content/pages/FAQ.rst]: Let's go ahead and use the shorter URL http://beaverbarcamp15.eventbrite.com

File: barcamp-theme/templates/base.html (hunk 1 above)
Comment (lucywyman) [inline barcamp-theme/templates/base.html]: Oh?  The information sheet for BBC15 was where I got the list from... 
https://docs.google.com/a/osuosl.org/document/d/1CWtkoamNHjX_515ux1EvVT__i7_kdW_DHxeZap41vDQ/edit

File: content/pages/FAQ.rst (hunk 3 above)
Comment (lucywyman) [inline content/pages/FAQ.rst]: I thought only the first session would be lightning talks. We would have the first session be just lightning talks, and then all other sessions be regular. Am I mistaken?

File: content/pages/info.rst [hunk 6]
```diff
@@ -6,7 +6,7 @@ WiFi
 ----
 
 We are using the OSU public wifi system. Look for **OSU_ACCESS** (choose local
-DB), account name is **BBC14**, password is **sleepyMandrill** (this account
+DB), account name is **BBC11**, password is **bossyPeccary** (this account
```
Comment (ramereth) [inline content/pages/info.rst]: So I think they did away with conference wifi addresses. We should update the instructions to having folks connect to the Visitor SSID and link this page: http://oregonstate.edu/helpdocs/network/wireless/getting-connected

File: content/pages/schedule.rst [hunk 7]
```diff
@@ -2,26 +2,23 @@ Schedule
...
-12:30pm - 1:30pm  Lunch
-1:30pm - 2:20pm   Session 3
-2:30pm - 3:20pm   Session 4
-3:30pm - 4:00pm   Snack Break
-4:00pm - 4:50pm   Session 5
-5:00pm - 5:50pm   Session 6
-================= ==================
+=================   ==================
+Time                Event
+9:00 – 10:00        Setup
```
Comment (ramereth) [inline content/pages/schedule.rst]: Please use the schedule found [here](https://docs.google.com/a/osuosl.org/document/d/1k5jwT-Ojt3DYBbf3wkgPgWXCUTW8X6miTBVh3aDUil8/edit) which if from last year.

File: content/pages/schedule.rst [hunk 8]
```diff
@@ -2,26 +2,23 @@ Schedule
...
+3:30 – 4:00         Snack Break
+4:00 – 4:50         Session 5
+5:00 – 5:50         Session 6
+6:00 – Closing      Remarks & cleanup
+=================   ==================
 
 Sessions
 --------
 
-.. raw:: html
```
Comment (ramereth) [inline content/pages/schedule.rst]: Why was this removed?

File: content/pages/schedule.rst (hunk 7 above)
Comment (lucywyman) [inline content/pages/schedule.rst]: I was using the one from the google doc: https://docs.google.com/a/osuosl.org/document/d/1CWtkoamNHjX_515ux1EvVT__i7_kdW_DHxeZap41vDQ/edit

Out of curiosity, why do we have the google doc if so much of the information on it is wrong?

File: barcamp-theme/templates/base.html (hunk 1 above)
Comment (ramereth) [inline barcamp-theme/templates/base.html]: Looks like they copied it from BBC11 and not from BBC14 (which seems to have only been visible to a few people). I just fixed that so the rest of the team can see it. We updated a lot of the information since BBC11. The one from BBC14 should be more accurate.

**Comment by lucywyman** [comment]
@ramereth Why do we have the google doc  https://docs.google.com/a/osuosl.org/document/d/1CWtkoamNHjX_515ux1EvVT__i7_kdW_DHxeZap41vDQ/edit if all of the information on it is wrong? That was what I based all of these changes off of.

File: content/pages/FAQ.rst (hunk 3 above)
Comment (ramereth) [inline content/pages/FAQ.rst]: My assumption the last time we talked about this was that the lightening talks are happening during the first session along with the other talks. Having it by itself is not going to go well with people.

**Comment by borcean** [comment]
@lucywyman Might want to gitignore .sass-cache :smiley: 
Keep all those binaries away

File: content/pages/FAQ.rst (hunk 3 above)
Comment (lucywyman) [inline content/pages/FAQ.rst]: Ok. I misunderstood. I'll modify that.

File: barcamp-theme/templates/base.html (hunk 1 above)
Comment (lucywyman) [inline barcamp-theme/templates/base.html]: Excellent, thank you!

**Comment by lucywyman** [comment]
@borcean Oh, grr! Thank you

**Comment by lucywyman** [comment]
@ramereth Ok, I think I've addressed everything you mentioned. Hopefully all the information is correct now!

File: barcamp-theme/templates/base.html [hunk 9]
```diff
@@ -61,7 +61,8 @@
 
 		    <p class="lead">Beaver BarCamp is an informal conference where everyone is encouraged to participate and the sessions are not predetermined. <a href="/pages/about.html"><em>Learn more</em></a></p>-->
 		    <h5>Event Sponsors</h5>
-        <p>Want to sponsor the event? Please contact info < at > beaverbarcamp.org</p>
+            <img class='sponsor' src='theme/img/osu-tag.png' />
+            <img class='sponsor' src='theme/img/mozilla.png' />
```
Comment (ramereth) [inline barcamp-theme/templates/base.html]: This image is broken.

File: content/lightning-talks.rst [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+:summary: Introducing Lightning talks
+
+This year we will be introducing Lightning talks to Beaver Barcamp!
+A lightning talk is a five-minute presentation on any given topic; 
+it's basically just a shorter version of the usual barcamp talk.  
+Instead of a keynote, our first session will be all lightning talks.
+You can come early and propose a topic to give a lightning talk on,
+or vote on other topics that you want to hear about. The most popular
+proposals will be chosen to give their presentations.  If you have
+any questions about this format, please email us at info<at>osuosl.org!
```
Comment (ramereth) [inline content/lightning-talks.rst]: Please change this to info@beaverbarcamp.org.

File: content/pages/info.rst [hunk 11]
```diff
@@ -5,9 +5,10 @@ Information
 WiFi
 ----
 
-We are using the OSU public wifi system. Look for **OSU_ACCESS** (choose local
-DB), account name is **BBC14**, password is **sleepyMandrill** (this account
-will work only for the weekend).
+We are using the OSU public wifi system. If you're an OSU student, we 
+suggest using OSU Secure and signing in with your ONID. If you're not
```
Comment (ramereth) [inline content/pages/info.rst]: You should have this be bolded and actually using the correct SSID name which is OSU_Secure.

File: content/pages/info.rst [hunk 12]
```diff
@@ -5,9 +5,10 @@ Information
...
+a student you can use the Visitor network.  Make sure you accept the
```
Comment (ramereth) [inline content/pages/info.rst]: Specifically state the SSID is Visitor and make that SSID name bold.

File: content/pages/schedule.rst [hunk 13]
```diff
@@ -23,5 +23,6 @@ Sessions
 --------
 
 .. raw:: html
+    
+     <iframe width='800' height='450' frameborder='0' src='https://docs.google.com/spreadsheet/pub?key=0Au0wOPQwuelMdEFFR0wzZHR5a3R3a3JOcDFFaGNjTGc&single=true&gid=9&output=html&widget=true'></iframe>
```
Comment (ramereth) [inline content/pages/schedule.rst]: I just made a blank BBC15 session sheet. Change ``gid=9`` to ``gid=10`` and it'll show up.

File: barcamp-theme/templates/base.html (hunk 9 above)
Comment (lucywyman) [inline barcamp-theme/templates/base.html]: Oops! Added image to the site.

**Comment by ramereth** [comment]
@lucywyman I see several pages (about, info, etc) where the images are broken if I run ``make html``. You might go through all of those and see why that's happening.

**Comment by lucywyman** [comment]
@ramereth I addressed all of the warnings that show up when I run :code:`make html` -- Let me know if any more show up for you!

**Comment by ramereth** [comment]
@lucywyman I'm still getting image errors when I look in the browser

**Comment by ramereth** [comment]
@lucywyman FWIW it works if I use the "dev server", but straight up ``make html`` is broken. I recommend you try accessing it via that. It **should** work with either.

**Comment by lucywyman** [comment]
@ramereth Can you pastebin your errors? I am successfully running both without any errors.

**Comment by ramereth** [comment]
@lucywyman its not on the console, build the site statically, open the site in your browser and you'll see image paths not working (they still assume / absolutely).

**Comment by lucywyman** [comment]
@ramereth Ok, I made the image paths relative so that they will work!  I also adjusted the font sizes so that they weren't so HUGE.  One thing, though, is that the current state of affairs is to have the links to the stylesheets be "{{ SITEULR }}/theme/css/style.css", which will work on every page in production but means that style changes don't show up in development.  I generally just change the siteurl in pelicanconf.py while developing and it isn't an issue, but it may be an issue we want to have a better/more long term fix for. I'm just not sure how to link to it without using relative links (as different pages will have different paths to get to it).

**Comment by lucywyman** [comment]
@ElijahCaine @jldugger I know you guys commented on font size -- you might take a look at the changes above and see if you're happy with them. I will be making more changes to theme in the future, so I'm really looking for a "this is good enough for now, right before barcamp" rather than a "this is good for the rest of ever and needs to be perfect".  I was hoping to get these changes merged last week, as time is of the essence!

**Comment by jldugger** [comment]
This one PR has gotten a little huge. It's a static site though, so +1.

But in the future if you're seeking approval, maybe consider how to break the changes into reviewable chunks.

**Comment by ramereth** [comment]
@lucywyman I'm still getting image issues :( ... even when I run ``make publish`` which is run by jenkins.

**Comment by lucywyman** [comment]
@ramereth there must be something up with your paths then.  I removed the entire output directory and remade the site and the images still show up for me using 'make publish', 'make html', and the development server (in that order, in case anyone is suspicious of caching). There should be a 'pages' directory and a 'theme' directory in your output.  You might also look at the 'publishconf.py', as RelativeURLS is set to false, though I kind of doubt that's the issue.  I'm honestly not sure what the issue could be, though :(  @jldugger Can you see the images?

**Comment by ramereth** [comment]
I am curious if anyone else is having the same problem as I have. I've noticed this both on my laptop and my workstation.

**Comment by ramereth** [comment]
@lucywyman I recommend you try a fresh checkout and clean build of the virtualenv on your end too.

**Comment by jldugger** [comment]
Lemme sort out probably unrelated build issues on my workstation and get back to you.

**Comment by jldugger** [comment]
Alright. Activating venvs helps.

Everything's coming up 200 on my browser after `make publish.` I don't think it solves the 'below the fold' issue, but IMO updating the schedule, recognizing sponsors and the like should be pushed out ASAP while we iterate or whatnot on more minor details.

**Comment by jldugger** [comment]
On further inspection, dev tools are lying to me. The sponsors images aren't loading, nor are they 404'ing.

**Comment by ramereth** [comment]
I see what's going on. The images seem correct with anything under ``pages/`` however the index, blog and lightening talk pages are broken with the sponsor pages. Probably because the link in the theme is hard coded for the images.

**Comment by jldugger** [comment]
Those are the two img elements I'm seeing load failed on.  If I remove the ../ they load.

**Comment by jldugger** [comment]
Yea, the sponsor logos are referenced are in the base.html template in a relative URL.

**Comment by ramereth** [comment]
@lucywyman there's probably a way to properly generate that using jinja templates with pelican within the theme instead of hard coding it like that.

**Comment by jldugger** [comment]
Part of the issue might be that make serve works, while using the filesystem does not. I suspect the python httpserver symlinks .. to . at the root to prevent malfeasance.

**Comment by lucywyman** [comment]
@ramereth Well, the about and sponsor pages are actually currently markdown pages, and I don't think they'll be processed as jinja templates. That's why when you said "Relative links", I added the '..'.  I'm experimenting with making the about page (and therefore other pages) just templates so that we can make use of jinja.

**Comment by lucywyman** [comment]
Quite honestly, I'm not sure what the optimal solution here is.  I would use {{ SITEURL }}, but then anything new -- even updates to the css -- won't show up unless we change it while developing, which seems like a poor solution.  Using "/theme/......." works with the development server, but not with 'make html', and using hard-coded relative paths only seems to work for my set up.  I googled around for some solutions, and came up empty handed -- it seems most people just use the "/theme/......" solution and only work with the development server.  My next step is to email Justin Mayer and ask if there's something I'm missing, but before I do that does anyone have any ideas which are better?  Personally, I feel like the image pathing on the current site obviously works as it is  (using the "/theme/......." -- look at the master branch to see what I'm talking about), and don't see any harm in leaving in that way.
